app/lib/dynamicRequire.js

```diff
@@ -133,4 +131,7 @@ module.exports = {
   // All Vue app components will be loaded via a vue chunk.  Eventually we will integrate webpack loading with Vue and
   // at that time we can remove this dynamic import and instead include SingletonAppVueComponentView in the main bundle
   'views/core/SingletonAppVueComponentView': function () { return import(/* webpackChunkName: "vue" */ 'views/core/SingletonAppVueComponentView').then(m => m.default) },
+
+  // Ozaria file split
+  'views/ozaria/play/level/PlayLevelView': function () { return import(/* webpackChunkName: "OzariaPlay" */ 'ozaria/views/play/level/PlayLevelView') },
```

Comment:
Its strange that `ozariaPlay` didnt work with a small `o`. OzariaUnitMap also has the same chunk name, I wonder how is that working.

Reply:
I spent an hour trying to figure this out. I have a hypothesis that it's because of [chunk size](https://hackernoon.com/the-100-correct-way-to-split-your-chunks-with-webpack-f8a9df5b7758)? In reality I have no idea. It just wouldn't work until I made this change.

I'm unsure how to proceed.

Reply:
It's ok to note it as a mystery and proceed if it works otherwise.